src/smart-components/request/request-actions.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Button } from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+
+import requestsMessages from '../../messages/requests.messages';
+import { isRequestStateActive } from '../../helpers/shared/helpers';
+
+const JustChildren = ({ children, className }) => <span className={ className }>{ children }</span>;
+
+JustChildren.propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string
+};
+
+const RequestActions = ({
+  denyLink,
+  approveLink,
+  commentLink,
+  request,
+  canApproveDeny,
+  canComment
+}) => {
+  const intl = useIntl();
+
+  const { id, state } = request;
+  const approveDenyAllowed = isRequestStateActive(state) && canApproveDeny;
+  const commentAllowed = isRequestStateActive(state) && canComment;
+
+  const ApproveDenyLink = approveDenyAllowed ? Link : JustChildren;
```

Comment:
the `JustChildren` component is a bit pointless. You can use syntax like this:
```jsx
const ApproveDenyLink = approveDenyAllowed ? Link : 'span';
```

Reply:
Actually, `JustChildren` component is catching all props other than `className`.

Reply:
Oh I did not think it was intentional